python/cuml/test/utils.py

```diff
@@ -216,9 +216,19 @@ def get_classes_from_package(package):
     return {k: v for dictionary in classes for k, v in dictionary.items()}
 
 
-def generate_random_labels(random_generation_lambda, seed=1234):
+def generate_random_labels(random_generation_lambda, seed=1234, as_cupy=False,
+                           with_numpy=False):
```

Comment:
Add a brief docstring to make this function quicker to understand. Also wouldn't it be easier to just always return the tuple and get rid of this? Seems like this two new parameters make the return values of the function more confusing than they need to be (am I getting cupy? with numpy? what if as_cupy is false and I ask with_numpy false, do I cause a segfault in the fabric of space and time?)

Reply:
Good point, I removed `with_numpy` but kept `as_cupy`. Not sure if you had an other idea in mind by removing both?